src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -3493,7 +3493,7 @@
     short-summary: Create a virtual network peering connection.
     long-summary: >
         To successfully peer two virtual networks this command must be called twice with
-        the values for --vnet-name and --remote-vnet-id reversed.
+        the values for --vnet-name and --remote-vnet reversed.
```

Comment:
You might consider a flag to do this automatically. Doesn't have to be this PR, but if you think it would improve the scenario it's worth considering.